backend/apps/owasp/models/project_health_metrics.py

```
@@ -156,6 +161,163 @@
+            ],
+            [
+                "Last Pull Request",
+                f"{metrics.last_pull_request_days}/{metrics.last_pull_request_days_requirement} days",
+            ],
+            [
+                "Last Release",
+                f"{metrics.last_release_days}/{metrics.last_release_days_requirement} days",
+            ],
+            [
+                "OWASP Page Last Update",
+                f"{metrics.owasp_page_last_update_days}/{metrics.owasp_page_last_update_days_requirement} days",
```

Ruff (0.12.2)

Line too long (112 > 99)

(E501)